pkg/vm/create_custom.go

```diff
@@ -16,7 +16,8 @@ import (
 	"github.com/ava-labs/avalanche-cli/pkg/ux"
 )
 
-func CreateCustomSidecar(
+func FillCustomSidecar(
```

Comment:
nit: I feel like CreateCustomSidecar was  fine as it was returning sidecar struct. as with this function it's not obvious that the result of this function is saved in the sidecar passed by pointer.
We should keep CreateCustomSIdecar model , just add a param with old sidecar to be modified and returned.
We can change a name to something like RecieveCustomSidecar or something better

Reply:
this would work like CreateCustomSIdecar if passed old sidecar is null